src/alert/alert.tsx

```diff
@@ -72,7 +118,14 @@ export const Alert = ({
                 </AlertIconWrapper>
             )}
             <TextContainer>
-                {children}
+                <TextWrapperContainer
+                    ref={contentResizeDetector.ref}
+                    $collapsedHeight={collapsedHeight}
+                    $showMore={showHiddenContent}
+                >
+                    {children}
```

Comment:
should set the contentResizeDetector on a static div, similar to the notification banner